test/runner/lib/delegation.py

```diff
@@ -168,6 +169,8 @@ def delegate_docker(args, exclude, require):
         '--docker-privileged': 0,
         '--docker-util': 1,
     }
+    if memory_limit:
+        options['--docker-memory'] = memory_limit
```

Comment:
These two lines are not needed, as there is no reason to filter out the option when delegating.